packages/v4/src/content/design-guidelines/components/navigation/navigation.md

```diff
@@ -29,27 +29,77 @@ Vertical navigation is hierarchical global navigation that displays navigation o
 * You have secondary navigation items (even if you have less than five primary navigation items)
 * You expect your application to be used on mobile devices
 
-#### Variations
+### Verical navigation variants
 
-**Simple vertical navigation**
+#### Simple vertical navigation
 
 Use a simple, single-level vertical navigation when you only have one level of navigation to display.
```

Comment:
I notice all the other sections start with "when you have ..." so maybe we should keep it consistent and change this to say "When you only have one level of navigation to display, use a simple, single-level vertical navigation"